projects/observability/src/pages/apis/api-detail/traces/api-trace-list.dashboard.ts

```diff
@@ -58,6 +58,19 @@ export const apiTraceListDashboard = {
           type: 'api-trace-navigation-handler'
         }
       },
+      {
+        type: 'table-widget-column',
```

Comment:
I think we're pulling two different features into the same PR here - can we split them up (error count on traces and error spans in waterfall)

Reply:
I have done that, please refer to : https://github.com/hypertrace/hypertrace-ui/pull/786